node/bft/src/gateway.rs

```diff
@@ -157,7 +154,7 @@ impl<N: Network> Gateway<N> {
             (Some(ip), _) => ip,
         };
         // Initialize the TCP stack.
-        let tcp = Tcp::new(Config::new(ip, Committee::<N>::MAX_COMMITTEE_SIZE));
+        let tcp = Tcp::new(Config::new(ip, N::MAX_CERTIFICATES.last().unwrap().1));
```

Comment:
Can `LATEST_MAX_CERTIFICATES` be used here for consistency? And since we return a Result anyway, can we properly handle the error?

Reply:
Is the error recoverable (i.e. occasionally expected)? If not, panicking may be appropriate to stop the node from continuing in an unexpected bad state?

Reply:
This operation is infallible, unless we ever intend to prune old entries in the list of maximums of certificates, so this change is mostly "cosmetic", and in this particular spot, painless :smiley:.

Reply:
It is not recoverable and not expected... don't feel *too* strongly either way